config/routes.rb

```diff
@@ -2,7 +2,13 @@
   resources :surveys do
     get 'results', on: :member
 
-    resources :questions
+    resources :conditionals
+    resources :questions do
+      member do
+        post :values
+        post :conditional_by_question
```

Comment:
If it's a `member` and you say just `conditional` it's clear that is the conditional by the `question_id` so it's not necessary to say `conditional_by_question`